presto-phoenix/src/test/java/io/prestosql/plugin/phoenix/TestPhoenixIntegrationSmokeTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.phoenix;
+
+import io.prestosql.tests.AbstractTestIntegrationSmokeTest;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.regex.Pattern;
+
+import static io.prestosql.plugin.phoenix.PhoenixQueryRunner.createPhoenixQueryRunner;
+import static org.testng.Assert.assertTrue;
+
+@Test
+public class TestPhoenixIntegrationSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    private TestingPhoenixServer testingPhoenixServer;
+
+    public TestPhoenixIntegrationSmokeTest()
+    {
+        this(TestingPhoenixServer.getInstance());
+    }
+
+    public TestPhoenixIntegrationSmokeTest(TestingPhoenixServer server)
+    {
+        super(() -> createPhoenixQueryRunner(server));
+        this.testingPhoenixServer = server;
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void destroy()
+    {
+        TestingPhoenixServer.shutDown();
+    }
+
+    @Test
+    public void testSchemaOperations()
+    {
+        assertUpdate("CREATE SCHEMA new_schema");
+
+        assertUpdate("CREATE TABLE new_schema.test (x bigint)");
+
+        assertQueryFails("DROP SCHEMA new_schema", Pattern.quote("Error while executing statement"));
```

Comment:
The quote call here should be a no-op since the pattern doesn't have any special regex characters.

Also, we should see if this can have a better error message, like `Cannot drop non-empty schema`